CONTRIBUTING.md

```diff
@@ -84,9 +87,9 @@ There is a build script `build/buildall` that automates the local build process.
 
 By default, it builds everything that is needed to create a distribution jar for all released (noSnapshots) Spark versions except for Databricks. Other profiles that you can pass using `--profile=<distribution profile>` include
 - `snapshots`
-- `minimumFeatureVersionMix` that currently includes 302, 311cdh, 312, 320 is recommended for catching incompatibilites already in the local development cycle
+- `minimumFeatureVersionMix` that currently includes 302, 311cdh, 312, 320 is recommended for catching incompatibilities already in the local development cycle
 
-For initial quick iterations we can use `--profile=<buildver>` to build a single-shim version. e.g., `-Dbuildver=320` for Spark 3.2.0
+For initial quick iterations we can use `--profile=<buildver>` to build a single-shim version. e.g., `--profile=301,321` for Spark 3.0.1 and 3.2.1
```

Comment:
text says single shim version but example shows multiple, I think we should stick with 1  in the example section.